packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -87,6 +87,8 @@ contract FaultDisputeGame is Clone, ISemver {
         address counteredBy;
     }
 
+    event Initialized(uint8 version);
```

Comment:
We should be able to remove this as long as we keep `IInitialized` in the ignore list, right?

Reply:
Yeah